src/components/Settings/components/WalletLimitsCard.vue

```diff
@@ -0,0 +1,394 @@
+                >
+                  <fa-icon icon="trash-alt"/>
+                </el-button>
+              </span>
+            </div>
+          </template>
+          <div
+            class="settings-item_row"
+            v-if="!accountLimits.length"
+          >
+            <span class="settings-item_row-key">You don't have any limits</span>
+          </div>
+        </div>
+      </div>
+    </el-card>
+    <el-dialog
+      data-cy="addWalletLimit"
+      title="Add wallet limit"
+      :visible.sync="addWalletLimitDialogVisible"
+      :show-close="false"
+      width="450px"
+      center
+    >
+      <div class="approval_form-desc">
+        <el-form class="dialog-form" :model="limitForm">
+          <el-form-item label="Wallets">
+            <el-select v-model="limitForm.assetId" class="fullwidth">
+              <el-option
+                v-for="wallet in wallets"
+                :key="wallet.name"
+                :label="`${wallet.name} (${wallet.asset.toUpperCase()})`"
+                :value="wallet.assetId">
+              </el-option>
+            </el-select>
+          </el-form-item>
+          <el-form-item label="Time">
+            <el-select v-model="limitForm.label" class="fullwidth">
+              <el-option
+                v-for="(timelimit, index) in timelimits"
+                :key="index"
+                :label="timelimit.label"
+                :value="timelimit.label">
+              </el-option>
+            </el-select>
+          </el-form-item>
+          <el-form-item label="Limit">
+            <el-input type="number" v-model="limitForm.amount" placeholder="0"/>
+          </el-form-item>
+        </el-form>
+      </div>
+      <div slot="footer" class="dialog-form_buttons-block">
+        <el-button
+          class="dialog-form_buttons action"
+          @click="onAddAssetLimit"
+          :loading="addingNewLimit"
+        >
+          Create
+        </el-button>
+        <el-button
+          class="dialog-form_buttons close"
+          @click="addWalletLimitDialogVisible = false"
+        >
+          Cancel
+        </el-button>
+      </div>
+    </el-dialog>
+    <el-dialog
+      data-cy="editWalletLimit"
+      title="Edit wallet limit"
+      :visible.sync="editWalletLimitDialogVisible"
+      :show-close="false"
+      width="450px"
+      center
+    >
+      <div class="approval_form-desc">
+        <el-form class="dialog-form" :model="editLimitForm">
+          <el-form-item label="Wallets">
+            <el-select v-model="editLimitForm.assetId" class="fullwidth" disabled>
+              <el-option
+                v-for="wallet in wallets"
+                :key="wallet.name"
+                :label="`${wallet.name} (${wallet.asset.toUpperCase()})`"
+                :value="wallet.assetId">
+              </el-option>
+            </el-select>
+          </el-form-item>
+          <el-form-item label="Time">
```

Comment:
I don't understand, this time only for one time or repeated period, we can change title or add placeholder with comment.